server/package.json

```diff
@@ -99,6 +100,7 @@
     "set-value": "^3.0.2",
     "sqlite3": "^4.0.6",
     "tracer": "^0.9.8",
+    "trend": "^0.3.0",
```

Comment:
So do you use this at the end ? I think this should be removed

Reply:
yes I miss to remove it , I fix it